README.md

```diff
@@ -53,14 +54,20 @@ Subsequent mutating operations are prohibited (only `deref`ing will work).
          :bucket "my_bucket"
          :key "0"
          :init {:x 1 :y 2})
+
+;; backed by Redis
+(duratom :redis-db
+         :db-config "any db-spec as understood by carmine"
+         :key-name "my:key"
+         :init {:x 1 :y 2})
 ```
 
 The initial-value <init> can be a concrete value (as show above), but also a no-arg fn or a delay. In any case, it may end up being completely ignored (i.e. if the underlying persistent storage is found to be non-empty).
 If you prefer passing arguments positionally, you can use the `file-atom`, `postgres-atom` & `s3-atom` equivalents.
 
 ## Custom :read & :write
 
-By default duratom stores plain EDN data (via `pr-str`). If that's good enough for your use too, you can skip this section. Otherwise, observe the following examples of utilising the `:rw` keyword for specifying [nippy](https://github.com/ptaoussanis/nippy) as the encoder/decoder of that EDN data:
+By default for most of the backends duratom stores plain EDN data (via `pr-str`). If that's good enough for your use too, you can skip this section. Otherwise, observe the following examples of utilising the `:rw` keyword for specifying [nippy](https://github.com/ptaoussanis/nippy) as the encoder/decoder of that EDN data:
```

Comment:
As I said, I'd prefer the default to be EDN - if `carmine` allows it of course.

Reply:
I changed it to default to edn.